console/ui/src/app/console.service.ts

```diff
@@ -839,14 +839,14 @@ export class ConsoleService {
   /** Delete (non-recorded) all user accounts. */
   deleteAccounts(auth_token: string): Observable<any> {
 		const urlPath = `/v2/console/account`;
```

Comment:
There's a few places where the generated file has indendation misalignment, this is likely due to a spaces vs tabs issues in the template.